src/components/Tooltip/Tooltip.driver.ts

```diff
@@ -0,0 +1,15 @@
+import {tooltipDriverFactory as coreTooltipDriverFactory} from 'wix-ui-core/dist/src/components/Tooltip/Tooltip.driver';
+
+export const toolTipDriverFactory = ({element, component, eventTrigger, wrapper}) => {
+  const driver = coreTooltipDriverFactory({element, component, eventTrigger, wrapper});
+  return {
```

Comment:
You should expose the entire driver outside

Reply:
done